cl/assets/templates/base.html

```diff
@@ -246,40 +246,118 @@ <h1>You did not supply the "private" variable to your template.
+              </ul>
             </li>
-            <li class="{% block navbar-oa %}inactive{% endblock %}" id="navbar-oa">
-              <a href="{% url "advanced_oa" %}" tabindex="104">Oral Arguments</a>
+            <li class="dropdown {% block navbar-oa %}inactive{% endblock %}" id="navbar-oa">
+              <a href="#"
+                 class="dropdown-toggle"
+                 tabindex="300"
+                 data-toggle="dropdown">Oral Arguments&nbsp;<span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                <li>
+                  <a href="{% url "advanced_oa" %}" tabindex="301">Search Oral Arguments</a>
+                </li>
+                <li>
+                  <a href="{% url "alert_help" %}#search-alerts" tabindex="302">Get Search Alerts</a>
+                </li>
+                <li>
+                  <a href="{% url "podcasts" %}" tabindex="303">Custom Podcasts</a>
+                </li>
+                <li>
+                  <a href="{% url "coverage_oa" %}" tabindex="304">About this Collection</a>
+                </li>
+                <li>
+                  <a href="{% url "oral_argument_api_help" %}" tabindex="305">API</a>
+                </li>
+              </ul>
+            </li>
+            <li class="dropdown {% block navbar-p %}inactive{% endblock %}" id="navbar-p">
+              <a href="#"
+                 class="dropdown-toggle"
+                 tabindex="400"
+                 data-toggle="dropdown">Judges&nbsp;<span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                <li>
+                  <a href="{% url "advanced_p" %}" tabindex="401">Search Judges</a>
+                </li>
+                <li>
+                  <a href="https://free.law/projects/judge-db" tabindex="404">About this Collection</a>
+                </li>
+                <li>
+                  <a href="{% url "judge_api_help" %}" tabindex="405">API</a>
+                </li>
+              </ul>
             </li>
-            <li class="{% block navbar-p %}inactive{% endblock %}" id="navbar-p">
-              <a href="{% url "advanced_p" %}" tabindex="105">Judges</a>
+            <li class="dropdown {% block navbar-fd %}inactive{% endblock %}" id="navbar-fd">
+              <a href="#"
+                 class="dropdown-toggle"
+                 tabindex="500"
+                 data-toggle="dropdown">Financial Disclosures&nbsp;<span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                <li>
+                  <a href="{% url "advanced_fd" %}" tabindex="501">Search Financial Disclosures</a>
+                </li>
+                <li>
+                  <a href="{% url "coverage_fds" %}" tabindex="502">About this Collection</a>
+                </li>
+                <li>
+                  <a href="{% url "financial_disclosures_api_help" %}" tabindex="503">API</a>
+                </li>
```

Comment:
this is great but its causing tests to fail - 

I think you can insert 

```@timeout_decorator.timeout(SELENIUM_TIMEOUT)
def test_disclosure_homepage(self) -> None:
    """Can we load disclosure homepage?"""
    self.browser.get(self.live_server_url)
    dropdown = self.browser.find_element(By.ID, "navbar-fd")
    dropdown.click()
    link = self.browser.find_element(By.LINK_TEXT, "Search Financial Disclosures")
    link.click()
``` into this particular test